Task2/Kuba/NoughtsAndCrosses/Game.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace NoughtsAndCrosses
+{
+    class Game
+    {
+        public IBoardCommunication interpreter;
+        public Game(IBoardCommunication interpreter)
+        {
+            this.interpreter = interpreter;
+        }
+
+        public void GameLoop(Player player1, Player player2, GameBoard board)
+        {
+            bool gameOver = false;
+            Player currentPlayer = player1;
+            while (!gameOver)
+            {
+                MakeMove(board, currentPlayer);
+                gameOver = GameIsOver(board, currentPlayer);
+                interpreter.ShowBoard(board);
+                if (gameOver) break;
```

Comment:
Agreed. Also, I think it's better to use something like GameStatus (i.e. active, tie, playerWon) in this while loop.